ArcSlider/ArcSlider.js

```diff
@@ -134,6 +142,14 @@ const ArcSliderBase = kind({
 		 */
 		radius: PropTypes.number,
 
+		/**
+		 * The current skin for this component.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		skin: PropTypes.string,
```

Comment:
Where do you use this prop?

Reply:
I deleted it, it was left over from when I tried another implementation.